packages/js-sync-with-portabl/src/lib/environments.ts

```diff
@@ -0,0 +1,14 @@
+export const environments = {
+  dev: {
```

Comment:
Realizing now we should potentially just remove dev and just leave production here for now --  dev will be an internal domain for Portabl where as we will have to create a sandbox environment for our users in the future which would still most likely be based off prod.